help/versions.md

```diff
@@ -1,59 +1,95 @@
-Submission Version Availability
-===============================
-
-In order to successfully complete a submission, a submitter agrees to
-our policy that all submitted works cannot be removed once publicly
-available. If an author needs to update the article after announcement,
-they may replace or withdraw the article, which creates a new version.
-This increments the version portion of the arXiv identifier. An example
-article with multiple versions is [arXiv:1710.02185](/abs/1710.02185).
-
-Making previous versions of submitted papers publicly available archives
-the historical record of research work. As such, it is a useful aid to
-peer review and verification of intellectual priority.
-
-What does arXiv mean by versions?
----------------------------------
-
-All submissions receive a "date stamp", recorded as the time the
-submitter clicks "Submit article" at the end of the submission process.
-Once a submission is made available, it receives a permanent [arXiv
-identifier](arxiv_identifier). Any replacement or withdrawal of that
-article generates a new version. This will increment its version number
-by one, with the most recent version always being displayed, even when
-the Abstract page is viewed without a version number at the end, such as
-the above example, `https://arxiv.org/abs/1710.02185`. For information
-on how to cite individual versions, please see the related
-[references](faq/references) FAQ page.
-
-What constitutes a version?
----------------------------
+# Submission Version Availability
+
+arXiv is intended as a historical collection of research works. Once made public, each version of a work is considered a permanent part of the scientific record and may not be removed.
+
+If an author needs to update the article after announcement, they may replace or withdraw the article, which creates a new version. This increments the version portion of the arXiv identifier. An example article with multiple versions is [arXiv:1710.02185](https://arxiv.org/abs/1710.02185).
+
+Making previous versions of submitted papers publicly available archives the historical record of research work. As such, it is a useful aid to peer review and verification of intellectual priority.
+
+
+## What does arXiv mean by versions?
+
+
+All submissions receive a "date stamp", recorded as the time the submitter clicks "Submit article" at the end of the submission process. Once a submission is made available, it receives a permanent  [arXiv identifier](arxiv_identifier).Any replacement or withdrawal of that article generates a new version. This will increment its version number by one, with the most recent version always being displayed, even when the Abstract page is viewed without a version number at the end, such as the above example, https://arxiv.org/abs/1710.02185. 

```

Comment:
```suggestion
All submissions receive a "date stamp", recorded as the time the submitter clicks "Submit article" at the end of the submission process. Once a submission is made available, it receives a permanent  [arXiv identifier](arxiv_identifier). Any replacement or withdrawal of that article generates a new version. This will increment its version number by one, with the most recent version always being displayed, even when the Abstract page is viewed without a version number at the end, such as the above example, https://arxiv.org/abs/1710.02185. 

```
typo.

Reply:
Good point about arXivID. I changed that to identifier to avoid lingo and ensure it translates OK.